### ISSUE TITLE
Issue with report files generation

### ISSUE THREAD

**Issue by AigerimKassym** [vdemichev/DiaNN#1057]
Dear Vadim,

I used DiaNN1.9 4 times and it worked perfectly, but from 5th analysis run it stopped generating report files. I repeated it twice, there were no errors in gui, it said report generated at the background but nothing was in the main output folder except quant files. 

I deleted and installed DiaNN again, didn't help

Best,
Aigerim

**Comment by vdemichev** [comment]
Hi Aigerim,

How does the log look like?

Best,
Vadim

**Comment by AigerimKassym** [comment]
![Untitled](https://github.com/vdemichev/DiaNN/assets/117651338/80efe4c1-405b-4ada-94c9-ab596d9be166)
![Untitled 0](https://github.com/vdemichev/DiaNN/assets/117651338/906b4ca0-53b3-400c-99f6-939fddadd650)
![Untitled 1](https://github.com/vdemichev/DiaNN/assets/117651338/f4efe97a-98fc-4989-8b8e-8c2976074418)

**Comment by vdemichev** [comment]
Hi Algerim,

Many thanks for letting me know. May I ask if you could please share the "spectral library.predicted.speclib" and the 12 .quant files for these raw data? I will then investigate the issue.

Best,
Vadim

**Comment by AigerimKassym** [comment]
Hi Vadim,
can I send via email? it is not enough space here for spectral library upload
Thanks!

**Comment by vdemichev** [comment]
Yes, please send to vadim dot demichev at gmail dot com.

**Comment by vdemichev** [comment]
Yes, please email! Email: my first name dot my last name at gmail dot com
